### PR TITLE
CVSL-436 survey links and rollout status page redesign

### PR DESCRIPTION
**Pull request by timharrison-moj**

### CODE REVIEW

File: helm_deploy/values-dev.yaml [hunk 1]
```diff
@@ -18,8 +18,8 @@ generic-service:
     PRISONER_SEARCH_API_URL: "https://prisoner-offender-search-dev.prison.service.justice.gov.uk"
     # PROBATION_SEARCH_API_URL: "https://probation-offender-search-dev.hmpps.service.justice.gov.uk"
     PRISON_REGISTER_API_URL: "https://prison-register-dev.hmpps.service.justice.gov.uk"
-    EXIT_SURVEY_LINK: "https://eu.surveymonkey.com/r/Preview/?sm=CQIWAIrnp68aIE1_2BNn3aRSTf_2BXWLpL8B9mtgPre1TKl4OznrnkajcbEz6SvRvXwn"
-    PHASE_BANNER_LINK: "https://eu.surveymonkey.com/r/Preview/?sm=CQIWAIrnp68aIE1_2BNn3aRSTf_2BXWLpL8B9mtgPre1TKl4OznrnkajcbEz6SvRvXwn"
+    EXIT_SURVEY_LINK: "https://eu.surveymonkey.com/r/V9JSHL3"
```
Comment (smcveigh941) [inline helm_deploy/values-dev.yaml:21]: Maybe just leave the dev one to the preview? Dont want to be sending feedback from dev!

File: server/views/pages/rollout-status.njk [hunk 2]
```diff
@@ -1,42 +1,36 @@
...
-    <div class="govuk-grid-row govuk-!-margin-bottom-4">
-        <div class="govuk-grid-column-three-quarters govuk-!-margin-left-2 govuk-!-margin-top-2">
-            <img id="rollout-status-map" alt="Map of pilot regions" src="/assets/images/rollout-status.png"/>
+            <p class="govuk-body">We'll let you know when it's coming and give you everything you need to use it.</p>
+
+            <h2 class="govuk-heading-m">About the Create and vary a licence service</h2>
+
+            <p class="govuk-body">It will replace both the PD1 and nDelius variation forms and processes for:</p>
+
+            <ul>
```
Comment (smcveigh941) [inline server/views/pages/rollout-status.njk:20]: Use lists like this

```
<p class="govuk-body">You can buy:</p>
<ul class="govuk-list govuk-list--bullet">
  <li>apples</li>
  <li>oranges</li>
  <li>pears</li>
</ul>
```

File: helm_deploy/values-dev.yaml (hunk 1 above)
Comment (timharrison-moj) [reply helm_deploy/values-dev.yaml:21]: I think being able to see how the survey looks in dev is useful, as this is where our people have access. We should probably collect an email address in the survey so we can differentiate real from test responses? Not sure on this. I did ask in slack and Ana was not bothered what showed in dev whilst Adam said make them all the same.

**Review by smcveigh941 — APPROVED** [review]
LGTM

**Pull request merged**